admin/tools.go

```diff
@@ -1,3 +1,4 @@
+//go:build tools
```

Comment:
Are these needed?

Reply:
No. I ran `go fmt ./...` by habit and it did that. I can undo those changes if they're a problem.